python/setup.py

```diff
@@ -225,7 +234,7 @@ def get_packages(self):
             "smart_open",
         ],
         "serve": ["uvicorn==0.16.0", "requests", "starlette", "fastapi", "aiorwlock"],
-        "tune": ["pandas", "tabulate", "tensorboardX>=1.9", "requests"],
+        "tune": [pandas_dep, "tabulate", "tensorboardX>=1.9", "requests"],
```

Comment:
hmm do we need to change this?

Reply:
Do we not want to have a uniform Pandas dependency version constraint? If not, then sure I can remove this.